app/lib/google_scraper_ruby/errors.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module GoogleScraperRuby
+  module Errors
+    class SearchServiceError < StandardError
+      def initialize(url:, error: nil)
+        @original_error = error
+        @url = url
+        super('')
+      end
+
+      def to_s
+        <<~ERR_MESSAGE
+          Failed to perform search request with URL: #{url}
+          - original_error: #{original_error}"
+        ERR_MESSAGE
+      end
+
+      private
+
+      attr_reader :original_error, :url
+    end
+
+      def initialize(url, error = nil)
```

Comment:
Seems like a rebase or copy/paste gone wrong as this code block seems to come from somewhere unexpected

Reply:
my bad. It was a rebase error, already fixed.